localisation/00_zuwc_united_localisation.csv

```diff
@@ -2885,6 +2737,109 @@ EVTOPT_MNM_4018;Cognition is what I live for.;;Erkenntnis ist das, wof
 String_Death_is_coming;Death is coming;;Der Tod kommt;;;;;;;;;;;x
 GetDinnerAndDrinks_dinner_and_drinks;dinner and plenty of drinks;;Abendessen und viele Getr�nke;;;;;;;;;;;x
 GetDinnerAndDrinks_meeting;a meeting;;ein Meeting;;;;;;;;;;;x
+ERA_CHAR_INFO_10019;Durotan, son of Garad and father of Go'el, was the chieftain of the Frostwolf Clan. Relations between the Frostwolf and the Blackrock clans were cold, even before the invasion of Azeroth, due to the Frostwolves' refusal to accept the demonic bargain and to drink Mannoroth's blood. When the Horde crossed into Azeroth, Durotan continued to express his opposition against the Horde's leadership, which pushed Gul'dan to exile the Frostwolf Clan.;Durotan, fils de Garad et p�re de Go'el, �tait le chef du clan Loup-de-givre. Les relations entre les Loup-de-givre et les clans Blackrock �taient glaciales, m�me avant l'invasion d'Azeroth, en raison du refus des Frostwolves d'accepter le march� d�monique et de boire le sang de Mannoroth. Lorsque la Horde passa en Azeroth, Durotan continua � exprimer son opposition � la direction de la Horde, ce qui poussa Gul'dan � exiler le clan Loup-de-givre.;Durotan, Sohn von Garad und Vater von Go'el, war der H�uptling des Frostwolf-Clans. Die Beziehungen zwischen dem Frostwolfclan und dem Blackrockclan waren schon vor der Invasion Azeroths eisig, da die Frostw�lfe sich weigerten, den d�monischen Pakt zu akzeptieren und das Blut von Mannoroth zu trinken. Als die Horde nach Azeroth kam, sprach Durotan sich weiterhin gegen die F�hrung der Horde aus, was Gul'dan veranlasste, den Frostwolfclan zu verbannen.;;;;;;;;;;;x
```

Comment:
why write in past tense here? wouldn't it make more sense to say that he IS the chieftain?

Reply:
This isn't actually my writing, except a few fixes. It was in the original one:
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/blob/463d21b3950c6d83aa8789dee55ce79fe2784fc0/localisation/00_zuwc_united_localisation.csv#L2060

Only reason it shows as "new" because I copied it to be bunched up with the other ones.